verify.go

```diff
@@ -0,0 +1,62 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const servicesPathPart = "Services"
+const verificationsPathPart = "Verifications"
+const verificationCheckPart = "VerificationCheck"
+
+type VerifyPhoneNumberService struct {
+	client *Client
+}
+
+type VerifyPhoneNumber struct {
+	Sid         string      `json:"sid"`
+	ServiceSid  string      `json:"service_sid"`
+	AccountSid  string      `json:"account_sid"`
+	To          string      `json:"to"`
+	Channel     string      `json:"channel"`
+	Status      string      `json:"status"`
+	Valid       bool        `json:"valid"`
+	Lookup      PhoneLookup `json:"lookup"`
+	Amount      string      `json:"amount"`
+	Payee       string      `json:"payee"`
+	DateCreated TwilioTime  `json:"date_created"`
+	DateUpdated TwilioTime  `json:"date_updated"`
+	URL         string      `json:"url"`
+}
+
+type CheckPhoneNumber struct {
+	Sid         string     `json:"sid"`
+	ServiceSid  string     `json:"service_sid"`
+	AccountSid  string     `json:"account_sid"`
+	To          string     `json:"to"`
+	Channel     string     `json:"channel"`
+	Status      string     `json:"status"`
+	Valid       bool       `json:"valid"`
+	Amount      string     `json:"amount"`
```

Comment:
types.NullString here, I think, based on the response.

Reply:
Fixed.